.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -547,8 +547,10 @@ AddType text/vtt                            vtt
     RewriteRule ^dedicated/architecture/additional_concepts/f5_big_ip\.html(.*) /dedicated/welcome/index.html$1 [NE,L,R=301]
     RewriteRule ^dedicated/architecture/additional_concepts/flannel\.html(.*) /dedicated/welcome/index.html$1 [NE,L,R=301]
 
-    # Redirects for the Dedicated restructure applied in https://github.com/openshift/openshift-docs/pull/41923 and https://github.com/openshift/openshift-docs/pull/43807
+    # Redirects for Dedicated service log content move, applied in https://github.com/openshift/openshift-docs/pull/47033
```

Comment:
@gaurav-nelson and @bscott-rh hi! Can you please review these redirects? Thank you!